docs/how-to/amplication-ai-examples.md

```diff
@@ -28,59 +28,105 @@ Can you create a new service for a blog application that will have posts, catego
 
-Is there anything else you would like to add or modify for your blog application?
+## Create New Modules
+
+Jovu can assist you in creating [new custom modules](/custom-types-and-actions#modules) for your service to help you better organize your service and encapsulate related functionality.
+
+**Question**
 ```
+In the Checkout module, I need to define the data structure for the information required when a user initiates a payment. Could you help me create a DTO that includes the payment amount, the currency, and the selected payment method?
+```
+
+![](./assets/amplication-ai/create-new-module.png)
 
-## Deploying an Amplication Project
+## Create New Data Transfer Objects (DTOs)
 
-Jovu can offer guidance on deploying your Amplication project to a production environment, ensuring your application is ready for real-world use.
+Jovu can [create new DTOs](/custom-types-and-actions#dtos-and-enums) to shape the data structures used for input and output in your service's APIs.
 
 **Question**
 ```
-Can you help me deploy my Amplication project to Docker Desktop?
+In the Checkout module, I need to define the data structure for the information required when a user initiates a payment. Could you help me c reate a DTO that includes the payment amount, the currecny, and the selected payment method?
 ```
 
-**Jovu's Response**
+![Create New Amplication DTOs](./assets/amplication-ai/create-new-dto.png)
+
+## Create Custom Actions For Your Modules
+
+Jovu will create [custom actions](/custom-types-and-actions#actions-1) (API endpoints) for your modules, so you can easily define the specific operations your service needs.
+
+**Question**
 ```
-Deploying your blog service to Docker Desktop involves several steps, primarily focusing on creating a Dockerfile for your service, building a Docker image, and then running that image as a container. Unfortunately, direct actions related to Docker or deployment processes outside of the Amplication platform are beyond the capabilities I can execute or automate directly from here. However, I'll guide you through the general steps:
+Can you help me create an Action to handle the approval or rejection of a comment for my BlogService and any necessary DTOs?
 ```
+![Create Custom Actions For Your Modules](./assets/amplication-ai/create-actions.png)
 
-<!-- >
-## Integrating a Plugin
+## Start New Projects
 
-Jovu can provide insights and general guidance on integrating third-party services, such as payment gateways or external APIs, into your Amplication project.
+Jovu can kickstart new projects by setting up the initial project structure and configuration based on your requirements.
 
-**Question:** I want to integrate a JSON Logger into my blog service. Can you guide me through the process of installing the plugin?
---->
+**Question**
+```
+I'm starting a new project fora client. Can you create a new project for me called AcmeCorp Website?
```

Comment:
"for a client" - typo

Reply:
In addition- this example is very simple. Nothing "exciting" here besides the "replacement" or creating a project manually. Let's try to find something more exciting (- will send a video that can help here)